client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -169,18 +173,32 @@ const PackageInfo = ( props ) => {
 			</div>
 
 			<div>
-				<FormFieldset className="packages-step__package-weight">
-				<FormLabel htmlFor={ `weight_${ packageId }` }>{ translate( 'Total Weight' ) }</FormLabel>
-					<FormTextInputWithAffixes
-						id={ `weight_${ packageId }` }
-						placeholder={ translate( '0' ) }
-						value={ pckg.weight || '' }
-						onChange={ onWeightChange }
-						isError={ Boolean( pckgErrors.weight ) }
-						type="number"
-						noWrap
-						suffix={ weightUnit } />
-					{ pckgErrors.weight && <FieldError text={ pckgErrors.weight } /> }
+				<FormFieldset>
+					<div className="packages-step__package-weight">
+						<FormLabel htmlFor={ `weight_${ packageId }` }>{ translate( 'Total Weight' ) }</FormLabel>
+						<FormTextInputWithAffixes
+							id={ `weight_${ packageId }` }
+							placeholder={ translate( '0' ) }
+							value={ pckg.weight || '' }
+							onChange={ onWeightChange }
+							isError={ Boolean( pckgErrors.weight ) }
+							type="number"
+							noWrap
+							suffix={ weightUnit } />
+						{ pckgErrors.weight && <FieldError text={ pckgErrors.weight } /> }
+					</div>
+					<div className="packages-step__package-signature">
+						<FormLabel htmlFor={ `signature_${ packageId }` }>{ translate( 'Require Signature' ) }</FormLabel>
+						<FormSelect
+							id={ `weight_${ packageId }` }
```

Comment:
Do we need `'weight'` here as a prefix? Maybe `signature` as a prefix will work better?

Reply:
Copy 🍝!